### ISSUE TITLE
Source selected even if you cancel

### ISSUE THREAD

**Issue by ljbellis** [chembl/curation-interface#147]
I drew a new compound and chose "Save all to ChEMBL"

I then chose Insert New Compound

I then choose Scientific Literature and Add New Document

I cancelled the Add new document

and then cancelled the Insert New Compound

However, the system kept hold of my chosen Source

![choose](https://cloud.githubusercontent.com/assets/9843928/9196448/6a0c4568-4023-11e5-8705-bdb9bdaf8a73.jpg)

It still would let me change this if I clicked on 'Save All To ChEMBL" again, but I think having it showing at the bottom of the page as chosen is misleading.

**Comment by mnowotka** [comment]
Yes, this is a very important topic. I would say, we should completely get rid of bottom bar and make document and source choices not persistent. The interface should always ask about them each time 'Save to ChEMBL' button is clicked.  But again, I'm open to any other suggestions.

**Comment by ljbellis** [comment]
Yes please - that would be great.
